int8 request
---
string map_name
string map_unique_id
